sys-apps/man-db/man-db-2.7.6.1-r3.ebuild

```diff
@@ -12,7 +12,7 @@ SRC_URI="mirror://nongnu/${PN}/${P}.tar.xz"
 LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="alpha amd64 arm ~arm64 hppa ia64 ~m68k ~mips ppc ppc64 ~s390 ~sh sparc x86 ~amd64-linux ~arm-linux ~x86-linux"
-IUSE="berkdb +gdbm +manpager nls selinux static-libs zlib"
+IUSE="berkdb +cron +gdbm +manpager nls selinux static-libs systemd zlib"
```

Comment:
Wait, the systemd timer won't be enabled automatically. The message shown in *pkg_postinst()* might tell you about the possibility to use a systemd timer instead of a cron job. So if you want to use a systemd timer instead of a cron job you are supposed to remove the default cron job.

At the moment, if you don't like an ebuild default, i.e. an automatically installed cron job, you are supposed to add an *INSTALL_MASK* for that particular file.